### ISSUE TITLE
Surrounding buildings does not use number of floors

### ISSUE THREAD

**Issue by lguilhermers** [architecture-building-systems/CityEnergyAnalyst#3219]
I thing the only feature requires by surroundings is its height (for solar radiation). Building floor seems to be used with floor height to calculate this parameter, but if height is taken directly from OSM (see my other issue), number of floors isn't even required.

Even in the case number of floor is used to calculate, they become independent parameters in the input editor. So I am in favor of removing this parameter from the input editor, leaving only height as an attribute. 
Please let me know if this makes sense.

**Comment by martin-mosteiro** [comment]
I don't mind either way. Even if number of floors is more intuitive for users, if they want to change the number of floors in the surrounding buildings they'll have to adjust the building height anyway, so it doesn't make a difference imo.

**Comment by lguilhermers** [comment]
Exactly, I think it is a parameter that doesn't impact any calculation and therefore, we don't need it. 
I will confirm if that is the case and, if so, remove it.
